device-types/Dell/PowerVault-MD1200.yml

```diff
@@ -0,0 +1,18 @@
+---
+manufacturer: Dell
+model: PowerVault MD 1200
+slug: dell-powervault-md1200
+part_number: powervault-md1200
+front_image: true
+rear_image: true
+comments: |
+  [PowerVault MD 1200](https://www.dell.com/en-in/work/shop/productdetailstxn/powervault-md1200)
+u_height: 2
+is_full_depth: true
+power-ports:
+  - name: PSU1
+    type: iec-60320-c14
+    maximum_draw: 600
+  - name: PSU2
+    type: iec-60320-c14
+    maximum_draw: 600
```

Comment:
These appear modular, are they? If so move their definitions to `module-bays` please